commands/openers.go

```diff
@@ -45,6 +45,10 @@ var projectLocalOpenCmd = &console.Command{
 	Usage:    "Open the local project in a browser",
 	Flags: []console.Flag{
 		dirFlag,
+		&console.StringFlag{
+			Name:  "webPath",
```

Comment:
This flag should probably be called `web-path` to match the convention of other flags, e.g. `--document-root`.

Reply:
Agreed

Reply:
maybe `--path` is enough here.